src/std_extensions/arithmetic/int_types.rs

```diff
@@ -30,37 +29,22 @@ fn int_custom_type(n: u8) -> CustomType {
 /// Integer type of a given bit width.
 /// Depending on the operation, the semantic interpretation may be unsigned integer, signed integer
 /// or bit string.
-pub fn int_type(n: u8) -> Type {
-    Type::new_extension(int_custom_type(n))
+pub fn int_type(width_power: u8) -> Type {
```

Comment:
Comment says this is "a given bit width". It certainly looks like it's just a `width` rather than a `width_power` whatever one of those is. (Same confusion as other comment, IOW)

Reply:
see  my comment above - I have not updated docs (or even the consts for that matter), just getting initial feedback on the idea for now